examples/axes_grid1/demo_axes_vbox_divider.py

```diff
@@ -0,0 +1,38 @@
+"""
```

Comment:
I suggest to merge this with "HBoxDivider demo". They are structurally equivalent. Having two separate examples blows up the number of examples and IMHO it's already diffdifficult to find something in there due to the sheer amount of examples.